main.cpp

```diff
@@ -96,40 +98,50 @@ int main()
     bool closeWindow(House::Kitchen myKitchen);
 };
 
+House::House()
+{
+    std::cout << "House Constructed" << std::endl;
+    std::cout << std::endl;
+}
+
+House::Kitchen::Kitchen()
+{
+    numKnives = 10;
+    std::cout << "Kitchen Constructed" << std::endl;
+    std::cout << std::endl;
+}
+
 void House::Kitchen::microwaveChicken( float microwaveTime)
 {
-    float instructions = 45.f;
-    microwaveTime = instructions;    
-    
     if( microwaveTime >= 45.f )
     {
-        std::cout << "Ready!" << std::endl;
+        std::cout << "Cooked for " << microwaveTime << " seconds:" << " Chicken cooked!" << std::endl;
     }
     else
     {
-        std::cout << "Not Ready!" << std::endl;
+        std::cout << "Cooked for " << microwaveTime << " seconds:" << " Chicken not finished cooking!" << std::endl;
     }
 }
 
-bool House::Kitchen::madeSalad( std::string lettuceBrand, bool includeTomatoes, std::string dressing)
+bool House::Kitchen::madeSalad(std::string lettuceBrand, bool includeTomatoes, std::string dressing)
 {
-    lettuceBrand = "Good Lettuce";
-
-    if( includeTomatoes == true && dressing == "Italian" )
+    if( includeTomatoes == true && dressing == "Italian" && lettuceBrand == "Lettuce!" )
     {
+        std::cout << "Made salad!" << std::endl;
         return true;
     }
     else
     {
+        std::cout << "Didn't make salad" << std::endl;
         return false;
     }
```

Comment:
If you find yourself writing code like this:
```
int Type::someFunction()
{
    if( someExpression )
    {
        someStatement();
        return aValue;
    } 
    else 
    {
        return anotherValue;
    }
}
```
You do not need the `else` block.  Here's why:
`return` statements are the last thing that **ever happens** in a function.
When the `if()` expression is evaluated as `true` and the block is entered, the `return aValue;` is the last line of code to be executed before the function exits.
but when the `if()` expression evaluates as `false` and the block is **NOT** entered, the next thing that happens is `return anotherValue;`
you have no code after the `else` block that needs executing, since your `else` block contains a `return` statement.
that means you do **NOT** need the `else` block, and it is just causing visual clutter.
```
int Type::someFunction()
{
    if( someExpression )
    {
        someStatement();
        return aValue;
    } 
    
    return anotherValue;
}
```
- [ ] revise